--- /tmp/ci/2372/prismasdwandevops/test/config/Roubaix OVH DC.yml
+++ /tmp/ci/2372/Roubaix OVH DC_controller.yml
@@ -1,8 +1,9 @@
 ---
 type: cloudgenix template
 version: 1.0
-# Created at 2022-08-07T09:01:51.096524Z
+# Created at 2022-08-29T22:30:26.183975Z
 # by system_2372@jeanlouis.prisma.sase
+# FROM_CLOUDBLADE
 sites v4.5:
   Roubaix OVH DC:
     address:
@@ -24,7 +25,7 @@
         - SPOKE
         cluster_insertion_mode: auto
         cluster_member_id:
-        connected: false
+        connected: true
         deployment_op:
         description:
         element_security_zones v2.0: []
